src/mocks/button.mock.json

```diff
@@ -0,0 +1,164 @@
+[
```

Comment:
These and the other mock files have data pulled via the little playground page below.